groups/infrastructure/main.tf

```diff
@@ -21,28 +21,32 @@ terraform {
 }
 
 module "ecs-cluster" {
-  source = "git@github.com:companieshouse/terraform-library-ecs-cluster.git?ref=1.1.4"
+  source = "git@github.com:companieshouse/terraform-modules//aws/ecs/ecs-cluster?ref=1.0.225"
```

Comment:
Looks like the latest module version is 1.0.228

(https://github.com/companieshouse/terraform-modules/tags)